filament/backend/src/opengl/OpenGLContext.cpp

```diff
@@ -449,6 +449,14 @@ void OpenGLContext::setDefaultState() noexcept {
         glClipControl(GL_LOWER_LEFT, GL_ZERO_TO_ONE);
 #elif defined(GL_EXT_clip_control)
         glClipControlEXT(GL_LOWER_LEFT_EXT, GL_ZERO_TO_ONE_EXT);
+#endif
+    }
+
+    if (ext.EXT_clip_cull_distance) {
+#if defined(BACKEND_OPENGL_VERSION_GL)
+        glEnable(GL_CLIP_DISTANCE0);
+#elif defined(GL_CLIP_DISTANCE0_EXT)
+        glEnable(GL_CLIP_DISTANCE0_EXT);
```

Comment:
I like to  `#define` the core defines to the ext ones instead. See, `gl_headers.h` where you'd do:

```#define GL_CLIP_DISTANCE0 GL_CLIP_DISTANCE0_EXT```

That's how I did it for other extensions.